ktoml-core/src/commonMain/kotlin/com/akuleshov7/ktoml/tree/TomlTable.kt

```diff
@@ -26,6 +28,69 @@ public abstract class TomlTable(
     public abstract var fullTableName: String
     public abstract var tablesList: List<String>
     public abstract val type: TableType
+
+    override fun write(
+        emitter: TomlEmitter,
+        config: TomlConfig,
+        multiline: Boolean
+    ) {
+        // Todo: Option to explicitly define super tables?
+        // Todo: Support dotted key-value pairs (i.e. a.b.c.d = 7)
+
+        val children = children
```

Comment:
this caching looks a little bit strange - may be use children from `this`?

Reply:
Yep, I have a habit of doing this to avoid accessing a field more than once, but I agree it looks weird. I'll change it